lib/node_modules/@stdlib/math/base/special/cflipsign/manifest.json

```diff
@@ -24,17 +26,56 @@
 	],
 	"confs": [
 		{
-			"src": [
-				"./src/main.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [],
-			"libpath": [],
-			"dependencies": [
-        "@stdlib/math/base/assert/is-negative-zero"
-      ]
+		  "task": "build",
+		  "src": [
+			"./src/main.c"
+		  ],
+		  "include": [
+			"./include"
+		  ],
+		  "libraries": [],
+		  "libpath": [],
+		  "dependencies": [
+			"@stdlib/math/base/napi/binary",
+			"@stdlib/complex/float64",
+			"@stdlib/complex/reim",
+			"@stdlib/math/base/assert/is-negative-zero",
+			"@stdlib/math/base/special/flipsign"
```

Comment:
Why is this needed for the implementation? I only see this being used in the benchmarks.